allensdk/api/cloud_cache/cloud_cache.py

```diff
@@ -226,6 +232,45 @@ def latest_downloaded_manifest_file(self) -> str:
             return ''
         return self._find_latest_file(self.list_all_downloaded_manifests())
 
+    def load_last_manifest(self):
+        """
+        If this Cache was used previously, load the last manifest
+        used in this cache. If this cache has never been used, load
+        the latest manifest.
+        """
+        if not self._manifest_last_used.exists():
+            self.load_latest_manifest()
+            return None
+
+        with open(self._manifest_last_used, 'r') as in_file:
+            to_load = in_file.read()
+
+        latest = self.latest_manifest_file
+
+        if to_load not in self.manifest_file_names:
+            msg = 'The manifest version recorded as last used '
+            msg += f'for this cache -- {to_load}-- '
+            msg += 'is not a valid manifest for this dataset. '
+            msg += f'Loading latest version -- {latest} -- '
+            msg += 'instead.'
+            warnings.warn(msg, UserWarning)
+            self.load_latest_manifest()
+            return None
+
+        if latest != to_load:
```

Comment:
do we want to propagate a suppression option (deaulted to not suppress) for this warning? For the case where, someone knows this, they will be running a cell that calls this many times over, and want to stop seeing it.
We could point annoyed users to this
https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings
or, add the suppression option and mention how to use the option in the warning.

Reply:
This only gets called when you instantiate `CloudCache` which only happens when you instantiate `VisualBehaviorOphysProjectCache`, so I think the user will only be encountering this warning once a session.

If you're still concerned about this, I guess I would vote for adding a suppression option to our code. Python's native warning suppression mechanism will catch warnings you didn't mean to. I am not a fan of that behavior.

Reply:
ok, we can pass